doc/api_conf.rst

```diff
@@ -44,6 +44,11 @@ Configurable settings of the :class:`dnf.Base` object are stored into a :class:`
 
     Debug messages output level, in the range 0 to 10. Default is 2.
 
+
+  .. attribute::defaultyes
+
+    Like ``assumeyes`` but will still prompt for confirmation. The default is ``False``.
```

Comment:
Ah apologies, I'm pretty new to the dnf codebase and I've not really used python before either. I had assumed that `api_conf` was an internal object reference so figured everything should be included.

What type of things are done with the Python API? If no user would ever be expected to interact with dnf during such a use case then I agree that this is not needed.